core/ui/SideBar/Open.tid

```diff
@@ -9,23 +9,29 @@ caption: {{$:/language/SideBar/Open/Caption}}
 <$action-listops $tiddler="$:/StoryList" $subfilter="+[insertbefore:currentTiddler<actionTiddler>]"/>
 \end
 
-<$list filter="[list[$:/StoryList]]" history="$:/HistoryList" storyview="pop">
-<div style="position: relative;">
-<$droppable actions=<<drop-actions>>>
-<div class="tc-droppable-placeholder">
-&nbsp;
+\define placeholder()
+\whitespace trim
```

Comment:
I think you don't need the `whitespace trim` here. It's only 1 line with no whitespace

Reply:
I think you're right, I'll try without